cross/duplicity/Makefile

```diff
@@ -1,15 +1,15 @@
 PKG_NAME = duplicity
-PKG_SHORT_VERS = 0.6
-PKG_VERS = $(PKG_SHORT_VERS).22
+PKG_SHORT_VERS = 0.7
+PKG_VERS = $(PKG_SHORT_VERS).03
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = https://launchpad.net/$(PKG_NAME)/$(PKG_SHORT_VERS)-series/$(PKG_VERS)/+download
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
-DEPENDS = cross/librsync
+DEPENDS = cross/librsync cross/netifaces cross/wrapt
```

Comment:
`cross/netifaces` and `cross/wrapt` should be removed here, as well as removed from `spk/Python/Makefile`. Both entries should be added to BUILD_DEPENDS of `spk/duplicity/Makefile`